src/citrine/resources/team.py

```diff
@@ -33,6 +33,52 @@ def __str__(self):
             .format(self.user.screen_name, self.actions, self.team.name)
 
 
+class TeamResourceIDs:
+    """
+    A Citrine Team Resource IDs class.
+
+    This class is used to list the unique IDs for specific resource
+    types published in a single team and therefore available to be pulled
+    in by all projects.
+
+    Parameters
+    ----------
+    team_id: str or uuid
+        ID of the team.
+    resource_type: str
+        The resource type to list, one of DATASET/MODULE/TABLE/TABLE_DEFINITION
+
+    """
+
+    _api_version = "v3"
+
+    def __init__(self,
+                 session: Session,
+                 team_id: Union[str, UUID],
+                 resource_type: str) -> None:
+        self.session = session
+        self.team_id = team_id
+        self.resource_type = resource_type
+
+    def _path(self) -> str:
+        return format_escaped_url(f'/teams/{str(self.team_id)}')
+
+    def _list_ids(self, action: str) -> List[str]:
+        query_params = {"domain": self._path(), "action": action}
+        return self.session.get_resource(f"/{self.resource_type}/authorized-ids",
+                                         params=query_params,
+                                         version=self._api_version)['ids']
+
+    def list_readable(self):
+        return self._list_ids(action="READ")
+
+    def list_writeable(self):
+        return self._list_ids(action="WRITE")
```

Comment:
This should be the module constant.
```suggestion
        return self._list_ids(action=WRITE)
```